### PR TITLE
allow for easier repo version setting

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: repos/katello-repos.spec [hunk 1]
```diff
@@ -1,3 +1,6 @@
+%global repo_version nightly
+%global repo_name    Nightly
```
Comment (stbenjam) [inline repos/katello-repos.spec]: I guess this still needs updating every release? We have the info below in the Version and Release, would my suggestion on the other PR not work?

```
REPO_VERSION=$(echo "puts '%{release}' =~ /nightly/ ? 'Nightly' : '%{version}'.split('.')[0..1].join('.')" | ruby)
sed -ie "s/@VERSION@/$REPO_VERSION/" $repofile
```

Comment (jlsherrill) [inline repos/katello-repos.spec]: oh right!  will adapt

Comment (jlsherrill) [inline repos/katello-repos.spec]: @stbenjam updated!

**Comment by stbenjam** [comment]
Awesome! ACK

**Pull request merged**